introduction.md

```diff
@@ -1,2 +1,5 @@
 *** INTRODUCTION TO VIKRANT BAINS ***
-My favorite code school path is iOS and Android
+My favorite code school path is iOS and Androi
+*Git
+*Javascriot
+*Ruby
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?